README.md

```diff
@@ -1,4 +1,4 @@
-# Outback Mate 3s Python library & command line interface
+# Outback Mate 3 & 3s Python library & command line interface
```

Comment:
I'm not sure this actually works on the Mate3. I think I just forgot the `s` when I wrote the original readme, so this has only ever been run against the Mate3s. Just mentioning it so you are aware, not sure any change is required. Maybe it *does* work with the Mate3 🤷‍♂️

Reply:
Mine's a Mate3 so we've got both covered!